app/models/concerns/document_date_concern.rb

```diff
@@ -3,13 +3,14 @@
 module DocumentDateConcern
   extend Blacklight::Solr::Document
 
-  FIELD_REGISTERED_DATE       = :registered_dttsim
-  FIELD_LAST_ACCESSIONED_DATE = :accessioned_latest_dttsi
-  FIELD_LAST_PUBLISHED_DATE   = :published_latest_dttsi
-  FIELD_LAST_SUBMITTED_DATE   = :submitted_latest_dttsi
-  FIELD_LAST_DEPOSITED_DATE   = :deposited_latest_dttsi
-  FIELD_LAST_MODIFIED_DATE    = :modified_latest_dttsi
-  FIELD_LAST_OPENED_DATE      = :opened_latest_dttsi
+  FIELD_REGISTERED_DATE           = :registered_dttsim
+  FIELD_LAST_ACCESSIONED_DATE     = :accessioned_latest_dttsi
+  FIELD_EARLIEST_ACCESSIONED_DATE = :accessioned_earliest_dttsi
```

Comment:
note...this is the only line added, everything else is just re-aligning the spacing